packages/dam-app-base/src/Editor/SortableComponent.tsx

```diff
@@ -121,164 +128,205 @@ const AdditionalDataDisplay = ({ additionalData }: AdditionalDataDisplayProps) =
+            </Flex>
+          </div>
+          <AdditionalDataDisplay additionalData={additionalData} />
+        </>
       );
-    }
-
-    if (additionalData) {
-      // if additional data is provided then the "more details" section will be visible
-      if (!url) {
-        return (
-          <>
-            <div>
-              <Flex justifyContent="center" title={alt}>
-                <FileIcon />
-              </Flex>
-            </div>
-            <AdditionalDataDisplay additionalData={additionalData} />
-          </>
-        );
-      } else {
-        return (
-          <>
-            <div>
-              <img src={url} alt={alt} title={alt} />
-            </div>
-            <Collapse isExpanded={isExpanded}>
-              {<AdditionalDataDisplay additionalData={additionalData} />}
-            </Collapse>
-            <TextLink
-              as="button"
-              onClick={() => setIsExpanded((currentIsExpanded) => !currentIsExpanded)}
-              className={styles.textlink}>
-              {isExpanded ? 'Hide details' : 'More details'}
-            </TextLink>
-          </>
-        );
-      }
     } else {
-      // display default asset card when no additional data is provided
-      if (!url) {
-        return (
-          <div className={styles.altTextContainer}>
-            <FileIcon />
-            <p className={styles.altTextDisplay} title={alt}>
-              {alt}
-            </p>
-          </div>
-        );
-      } else {
-        return (
+      return (
+        <>
           <div>
             <img src={url} alt={alt} title={alt} />
           </div>
-        );
-      }
+          <Collapse isExpanded={isExpanded}>
+            {<AdditionalDataDisplay additionalData={additionalData} />}
+          </Collapse>
+          <TextLink
+            as="button"
+            onClick={() => setIsExpanded((currentIsExpanded) => !currentIsExpanded)}
+            className={styles.textlink}>
+            {isExpanded ? 'Hide details' : 'More details'}
+          </TextLink>
+        </>
+      );
+    }
+  } else {
+    // display default asset card when no additional data is provided
+    if (!url) {
+      return (
+        <div className={styles.altTextContainer}>
+          <FileIcon />
+          <p className={styles.altTextDisplay} title={alt}>
+            {alt}
+          </p>
+        </div>
+      );
+    } else {
+      return (
+        <div>
+          <img src={url} alt={alt} title={alt} />
+        </div>
+      );
     }
   }
-);
+};
+
+const SortableItem = ({
+  url,
+  alt,
+  disabled,
+  onDelete,
+  additionalData,
+  uniqueId,
+}: SortableElementProps) => {
+  const { attributes, listeners, setNodeRef, transform, transition } = useSortable({
+    id: uniqueId,
+  });
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
 
-const SortableItem = SortableElement<SortableElementProps>((props: SortableElementProps) => {
```

Comment:
the `SortableItem` component also stays much the same.  Just swapping out `react-sortable-hoc`'s `SortableElement` wrapper for `dndkit`'s `useSortable()` hook.